sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -980,5 +1010,7 @@ SELECT
   map_sum_aggregates[OFFSET(29)].map AS scalar_parent_urlbar_picked_topsite_sum,
   map_sum_aggregates[OFFSET(30)].map AS scalar_parent_urlbar_picked_unknown_sum,
   map_sum_aggregates[OFFSET(31)].map AS scalar_parent_urlbar_picked_visiturl_sum,
+  map_sum_aggregates[OFFSET(32)].map AS search_with_ads,
+  map_sum_aggregates[OFFSET(33)].map AS ad_clicks,
```

Comment:
I chose these names without `scalar_parent_...` to match the existing `search_with_ads_count_all` and `ad_clicks_count_all` columns